addons/stock/views/stock_picking_views.xml

```diff
@@ -369,6 +369,7 @@
                                     <field name="is_initial_demand_editable" invisible="1"/>
                                     <field name="is_quantity_done_editable" invisible="1"/>
                                     <field name="product_uom_qty" string="Demand" attrs="{'column_invisible': [('parent.immediate_transfer', '=', True)], 'readonly': ['|', ('is_initial_demand_editable', '=', False), '&amp;', '&amp;', ('show_operations', '=', True), ('is_locked', '=', True), ('is_initial_demand_editable', '=', False)]}"/>
+                                    <button type="object" name="action_product_forecast_report" attrs="{'invisible': [('parent.picking_type_code', '=', 'outgoing'), ('parent.state','!=', 'done')]}" icon="fa-area-chart"/>
```

Comment:
This button is already present in the widget `forecast_availability`, then in some situation it is possible to have both forecasted button, it is ugly no and should be merge ?

Reply:
Ok, I just get it. forget my this comment. 
But we should merge it with the widget of `forecast_availability`